components/shared/azure-pipelines.yml

```diff
@@ -1,11 +1,18 @@
 trigger:
-- master
+  branches:
+    include:
+    - repo-restructure
+    - master
+  paths:
+    exclude:
+    - stream-provider
+    - stream-consumer
 
 pool:
   vmImage: 'ubuntu-latest'
 
 steps:
-- script: docker build . -t bengreenier/browserd:$(Build.BuildId) -t bengreenier/browserd:latest
+- script: docker build . -f components/shared/Dockerfile 
 - task: Docker@2
```

Comment:
this deploys the docker container, i don't think we want to deploy the "shared" container, and definitely not to "browserd" - can we remove?